susemanager-utils/susemanager-sls/salt/bootstrap/init.sls

```diff
@@ -145,10 +145,10 @@ salt-minion-package:
     - require:
       - file: bootstrap_repo
 
-{# These two dependencies are only needed on DEB based distros with Python version < 3.7 #}
-{# We cannot make these packages as hard depedendencies for Salt package because this is only needed for Ubuntu 18.04 #}
+{# These dependencies are only needed on DEB based distros, running Salt 3004, with Python version < 3.7 #}
+{# We cannot make these packages as hard depedendencies for Salt package because this is only needed for Ubuntu 18.04 and Debian 9 #}
```

Comment:
```suggestion
{# We cannot make these packages as hard depedendencies for Salt package because this is only needed for Ubuntu 16.04 and Debian 9 #}
```

Reply:
My comment is not right either. Ubuntu 18.04 has python 3.6